src/components/screens/on-call-calendar-screen.tsx

```diff
@@ -101,31 +105,30 @@ const OnCallCalendarScreen = () => {
   };
 
   /**
- * Renders the current week's on call person if they exist
- */
-const renderCurrentOnCall = () => {
-  const isValidPerson = onCallPerson !== "No person on call" && onCallPerson !== null;
-
-  return (
-    <Typography
-      sx={{
-        textAlign: "center",
-        margin: 2,
-        color: isValidPerson ? "#5acc31" : red[700],
-        fontWeight: "bold",
-      }}
-    >
-      {isValidPerson ? (
-        <>
-          {strings.oncall.onCallPersonExists}
-          <b style={{ color: "black", fontWeight: "bold" }}> {onCallPerson}</b>
-        </>
-      ) : (
-        strings.oncall.noOnCallPerson
-      )}
-    </Typography>
-  );
-};
+   * Renders the current week's on call person if they exist
+   */
+  const renderCurrentOnCall = () => {
+    const isValidPerson = onCallPerson !== null;
+    return (
+      <Typography
+        sx={{
+          textAlign: "center",
+          margin: 2,
+          color: isValidPerson ? "#5acc31" : red[700],
+          fontWeight: "bold",
+        }}
+      >
+        {isValidPerson ? (
+          <>
+            {strings.oncall.onCallPersonExists}
+            <b style={{ color: "black", fontWeight: "bold" }}> {onCallPerson}</b>
```

Comment:
Don't use HTML, we have the component library, Typography can be used here I'm quite sure, although then we have nested typographies! Perhaps consider replacing the top level typography with a layout component e.g. box, container or stack. And then the strings themselves can be all placed in typographies. 
This should give more control over layout as well